rhel8/CMakeLists.txt

```diff
@@ -25,9 +25,10 @@ ssg_build_html_nistrefs_table(${PRODUCT} "ospp")
 
 ssg_build_html_cce_table(${PRODUCT})
 
+ssg_build_html_srgmap_tables(${PRODUCT} ${DISA_SRG_TYPE})
+
+ssg_build_html_stig_tables(${PRODUCT} "stig")
```

Comment:
```suggestion
ssg_build_html_stig_tables(${PRODUCT} "ospp")
```

Reply:
@shawndwells oooppsss... wasn't supposed to be there. Removed.